src/go/k8s/tests/e2e/update-image-tls/00-assert.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: apps/v1
```

Comment:
remove this

Reply:
ping

Reply:
wait, how did this not get removed :D thanks